pkg/daemon/daemon.go

```diff
@@ -142,6 +144,29 @@ func NewUnikDaemon(config config.DaemonConfig) (*UnikDaemon, error) {
 		break
 	}
 
+	for _, openstackConfig := range config.Providers.Openstack {
+		openstack.MergeConfWithEnv(&openstackConfig)
+
+		// Mask password prior logging to console.
+		orig_pass := openstackConfig.Password
+		openstackConfig.Password = "<password>"
+		logrus.Infof("Bootstrapping provider %s with config %v", openstack_provider, openstackConfig)
+		openstackConfig.Password = orig_pass
+
+		p, err := openstack.NewOpenstackProvider(openstackConfig)
+		if err != nil {
+			return nil, errors.New("initializing openstack provider", err)
+		}
+		s, err := state.BasicStateFromFile(openstack.OpenstackStateFile())
+		if err != nil {
+			logrus.WithError(err).Warnf("failed to read openstack state file at %s, creating blank openstack state", openstack.OpenstackStateFile())
+			s = state.NewBasicState(openstack.OpenstackStateFile())
+		}
```

Comment:
Currently, this problem is addressed in README of every provider, e.g. [AWS](https://github.com/emc-advanced-dev/unik/blob/master/docs/providers/aws.md). We tell user that
```
If UniK gets into a bad state (i.e. you manually remove a file or AWS VM), 
you should manually edit the $HOME/.unik/aws/state.json file to remove 
the instance that no longer exists. UniK will eventually become 
self-correcting to deal with disruptions in the state.
```